libopenage/unit/action.h

```diff
@@ -419,6 +420,11 @@ class RepairAction: public TargetAction {
 private:
 	bool complete;
 
+	float time;
+	float time_left;
+
+	ResourceBundle cost; // calculate and store the cost in the constructor
```

Comment:
you can write `/** stores the cost of the repair */` or whatever is more suitable above it so it's parsed by doxygen, too.